### ISSUE TITLE
Game crashes after loading into the world

### ISSUE THREAD

**Issue by 698969** [nicguzzo/wands#4]
MC version: 1.16.5
Mod version: 1.4.5 (on Fabric 0.11.3)

java.lang.NullPointerException: Unexpected error
	at net.minecraft.class_1810.redirect$dgm000$applyMiningSpeedMultiplierMultiplier(class_1810.java:524)
	at net.minecraft.class_1810.method_7865(class_1810.java:143)
	at net.nicguzzo.ICompatModImpl.can_destroy(ICompatModImpl.java:288)
	at net.nicguzzo.common.WandsBaseRenderer.render(WandsBaseRenderer.java:102)
	at net.nicguzzo.WandsClientMod.render(WandsClientMod.java:168)
	at net.minecraft.class_863.handler$cec000$render(class_863.java:545)
	at net.minecraft.class_863.method_23099(class_863.java:168)
	at net.minecraft.class_761.method_22710(class_761.java:1340)
	at net.minecraft.class_757.method_3188(class_757.java:727)
	at net.minecraft.class_757.method_3192(class_757.java:546)
	at net.minecraft.class_310.method_1523(class_310.java:1048)
	at net.minecraft.class_310.method_1514(class_310.java:681)
	at net.minecraft.client.main.Main.main(Main.java:215)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226)
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)

I can't seem to reproduce the error consistently, but when it does crash I have the wand selected in my hotbar when I load into the world.

**Comment by nicguzzo** [comment]
check 1.4.7, I couldn't repdroduce the crash, but I think I patched the potential bug.

**Comment by 698969** [comment]
I tried to load the same world with the 1.4.7 version and got pretty much the same stacktrace but with different line numbers on 2 lines. 

java.lang.NullPointerException: Unexpected error
at net.minecraft.class_1810.redirect$dgm000$applyMiningSpeedMultiplierMultiplier(class_1810.java:524)
at net.minecraft.class_1810.method_7865(class_1810.java:143)
at net.nicguzzo.ICompatModImpl.can_destroy(ICompatModImpl.java:**290**)
at net.nicguzzo.common.WandsBaseRenderer.render(WandsBaseRenderer.java:**104**) ..........

So I guess the bug is still there, I'll test it on a new world just to be sure and let you know.

Some further info, on 1.4.5 I disabled the mod when it crashed and re enabled it, the previous wands were gone and I crafted some new ones. The new ones worked fine until I used one up to about half of its durability. But when I tried to put the wand from my inventory back into my selected hotbar slot the crash appeared again. So, it doesn't just happen when loading in but could also happen in the middle of a game.

And if I remember correctly, the wand was also nearly halfway used when the crash happened the first time.

The crash report mentions the render thread, it's not a conflict with sodium or indium is it?
